test/e2e/networking_perf.go

```diff
@@ -0,0 +1,238 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+// Tests network performance using iperf or other containers.
+import (
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+	"time"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/util/intstr"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// Declared as Flakey since it has not been proven to run in parallel on small nodes or slow networks in CI
+var _ = Describe("Networking IPerf [Experimental] [Networking] [Flaky]", func() {
```

Comment:
to be clear, this test isn't flaky by definition, but it has the potential to be in an unstable cloud, and we dont wanna destabilize the baseline CI at this time when so much is going on. hence the label.